hail/python/hail/backend/backend.py

```diff
@@ -325,14 +328,12 @@ def execute(self, ir, timed=False):
             resp_json = resp.json()
             raise FatalError(resp_json['message'])
         resp.raise_for_status()
-
         resp_json = resp.json()
         typ = dtype(resp_json['type'])
-        result = json.loads(resp_json['result'])
-        value = typ._from_json(result['value'])
-        timings = result['timings']
+        value = typ._convert_from_json_na(resp_json['value'])
+        # FIXME put back timings
```

Comment:
We should put back the timings?

Reply:
I'd have to change the API to send timings back to the client.  I'd like to hold off on this while I'm formulating the larger plan.  In particular, I'm thinking about server-side dev tools to investigate how pipelines executed in the service spend time, view IR after optimization and lowering, etc.  These would be visible to us, but not users an would not be part of the protocol.  I hope this will get fleshed out over the next couple of months and the service gets filled out.